COMPILING.md

```diff
@@ -0,0 +1,62 @@
+# Compiling GoZen
+
+This guide will help you compile the GoZen gdextension.
+
+## Step 1: Installing dependencies
+
+### Linux
+
+### Windows
+
+- Install Msys2 from [www.msys2.org](https://www.msys2.org/)
+- Add `path\to\msys64\mingw64\bin` and `path\to\msys64\usr\bin` to the `Path` environment variable. The default installation path is `C:\msys64\`
+
+- Update all packages by executing the following command in the terminal:
+
+    ```bash
+    pacman -Suy
+    ```
+
+    **Note:** In some cases pacman will prompt you to close all terminals. After confiming start a new terminal and re-run the above command.
+
+- Install the required dependencies using the following command:
+
+    ```bash
+    pacman -S mingw-w64-cross-binutils mingw-w64-x86_64-toolchain mingw-w64-x86_64-scons mingw-w64-x86_64-yasm diffutils make
+    ```
+
+## Step 2: Cloning repositories
+
+- Open a terminal in the desired directory and clone the Gozen repository
```

Comment:
It should be GoZen (with the capital letter Z), not Gozen.

Reply:
Nice catch!

Reply:
Added this to the review in which I also added the info for dependencies for Linux.